requirements.txt

```diff
@@ -1,3 +1,4 @@
+requests
```

Comment:
Could you please ping a specific version of requests here? a requirements.txt is aimed to be extremly specific, so anyone could reproduce an exact environment while working (contrary to a setup.py install_requirements which should be as broad as possible to keep compatibility with other installed packages).

Reply:
fix d05a5e66b1eae5c390ce58ecc96f8916bf1c51e2